docs/contributor/03-60-regions-configuration.md

```diff
@@ -0,0 +1,47 @@
+# Regions and Zones Configuration
+
+Most Kyma plans require specifying a region in the request body as one of the provisioning parameters. The regions' configuration allows you to control which regions are available for the Kyma plan and the platform region. Additionally, every region has a different set of zones.
+The configuration is used to validate the region specified in the request body. Additionally, those regions are used to generate schemas for the catalog endpoint.
+
+## Allowed Regions
+
+The `plansConfiguration` section of the `values.yaml` file contains a list of plans. Every plan has a platform region with allowed regions. To avoid specifying all possible platform regions, define `default` values, for example:
+
+```yaml
+
+plansConfiguration:
+  aws:
+    cf-eu11:
+      - eu-central-1
+    default:
+      - us-east-1
+      - us-west-2
+```
+
+The above configuration means that the `cf-eu11` platform region can only use the `eu-central-1` provider region, while all other platform regions can use the `us-east-1` and `us-west-2` provider regions. 
+Trial and free plans are not listed in the `plansConfiguration` section. The list of allowed regions for free is calculated in the code. The trial plan does not support region parameter.
```

Comment:
```suggestion
The trial and free plans are not listed in the `plansConfiguration` section. The list of allowed regions for the free plan is calculated in the code. The trial plan does not support the **region** parameter.
```

Reply:
In the code sample below, it is `regions`, so should it be "the **regions** parameter" (plural)?